keyboards/alhenkb/numpad/info.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Are these separate PCBs? 

Because they appear to be wired up identically, and could use the same config files for both, but separate layouts. (eg, move the layout numpad into the same json as the ortho layout).

Reply:
Fixed, thank you for the advice!